### ISSUE TITLE
StyleComponent build error

### ISSUE THREAD

**Issue by derevandal** [Developmint/nuxt-svg-loader#32]
After 1.0.1 release, my builds is failing.
```html
<tecnospeed-logo
  :class="!isFixed ? 'svgToWhite' : ''"
  width="180"
  alt="Logo da TecnoSpeed"
  class="img-fluid" />
```
``` js
import TecnospeedLogo from '@/assets/images/tecnospeed.svg'
export default {
  name: 'CommonNavbar',
  components: {
    /* ... */
    TecnospeedLogo
  }
}
```
``` bash
8:40:24 PM: ERROR in ./assets/images/tecnospeed.svg?vue&type=script&lang=js& (./node_modules/babel-loader/lib??ref--2-0!./node_modules/vue-loader/lib!./node_modules/svg-to-vue-component/loader.js!./assets/images/tecnospeed.svg?vue&type=script&lang=js&)
8:40:24 PM: Module build failed (from ./node_modules/babel-loader/lib/index.js):
8:40:24 PM: SyntaxError: /opt/build/repo/assets/images/tecnospeed.svg: Unexpected token (3:17)
8:40:24 PM:   1 | //
8:40:24 PM:   2 |
8:40:24 PM: > 3 | import Component "/opt/build/repo/node_modules/svg-to-vue-component/lib/StyleComponent.js"
8:40:25 PM:     |                  ^
8:40:25 PM:   4 | export default {
8:40:25 PM:   5 |   components: {
8:40:25 PM:   6 |     'svg2vue-style': Component
8:40:25 PM:     at _class.raise (/opt/build/repo/node_modules/@babel/parser/lib/index.js:4051:15)
8:40:25 PM:     at _class.unexpected (/opt/build/repo/node_modules/@babel/parser/lib/index.js:5382:16)
8:40:25 PM:     at _class.expectContextual (/opt/build/repo/node_modules/@babel/parser/lib/index.js:5350:41)
8:40:25 PM:     at _class.parseImport (/opt/build/repo/node_modules/@babel/parser/lib/index.js:8823:12)
8:40:25 PM:     at _class.parseStatementContent (/opt/build/repo/node_modules/@babel/parser/lib/index.js:7586:27)
8:40:25 PM:     at _class.parseStatement (/opt/build/repo/node_modules/@babel/parser/lib/index.js:7505:17)
8:40:25 PM:     at _class.parseBlockOrModuleBlockBody (/opt/build/repo/node_modules/@babel/parser/lib/index.js:8073:23)
8:40:25 PM:     at _class.parseBlockBody (/opt/build/repo/node_modules/@babel/parser/lib/index.js:8060:10)
8:40:25 PM:     at _class.parseTopLevel (/opt/build/repo/node_modules/@babel/parser/lib/index.js:7470:10)
8:40:25 PM:     at _class.parse (/opt/build/repo/node_modules/@babel/parser/lib/index.js:8915:17)
8:40:25 PM:     at parse (/opt/build/repo/node_modules/@babel/parser/lib/index.js:10946:38)
8:40:25 PM:     at parser (/opt/build/repo/node_modules/@babel/core/lib/transformation/normalize-file.js:170:34)
8:40:25 PM:     at normalizeFile (/opt/build/repo/node_modules/@babel/core/lib/transformation/normalize-file.js:138:11)
8:40:25 PM:     at runSync (/opt/build/repo/node_modules/@babel/core/lib/transformation/index.js:44:43)
8:40:25 PM:     at runAsync (/opt/build/repo/node_modules/@babel/core/lib/transformation/index.js:35:14)
8:40:25 PM:     at process.nextTick (/opt/build/repo/node_modules/@babel/core/lib/transform.js:34:34)
8:40:25 PM:  @ ./assets/images/tecnospeed.svg?vue&type=script&lang=js& 1:0-210 1:226-229 1:231-438 1:231-438
8:40:25 PM:  @ ./assets/images/tecnospeed.svg
8:40:25 PM:  @ ./node_modules/babel-loader/lib??ref--2-0!./node_modules/vue-loader/lib??vue-loader-options!./components/common/navbar.vue?vue&type=script&lang=js&
8:40:25 PM:  @ ./components/common/navbar.vue?vue&type=script&lang=js&
8:40:25 PM:  @ ./components/common/navbar.vue
8:40:25 PM:  @ ./node_modules/babel-loader/lib??ref--2-0!./node_modules/vue-loader/lib??vue-loader-options!./layouts/error.vue?vue&type=script&lang=js&
8:40:25 PM:  @ ./layouts/error.vue?vue&type=script&lang=js&
8:40:25 PM:  @ ./layouts/error.vue
8:40:25 PM:  @ ./.nuxt/index.js
8:40:25 PM:  @ ./.nuxt/client.js
8:40:25 PM:  @ multi ./.nuxt/client.js
```

**Comment by manniL** [comment]
@derevandal Thanks for the report! Could you file an issue in https://github.com/egoist/svg-to-vue-component as that's the code responsible for the conversion? ☺️

**Comment by derevandal** [comment]
I do this ;) Thanks

**Comment by derevandal** [comment]
Fixed on `svg-to-vue-component@0.3.2`[#17](https://github.com/egoist/svg-to-vue-component/issues/17)
